api_docgen/odoc/Makefile

```diff
@@ -0,0 +1,188 @@
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*             Florian Angeletti, projet Cambium, Inria Paris             *
+#*                                                                        *
+#*   Copyright 2020 Institut National de Recherche en Informatique et     *
+#*     en Automatique.                                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+include $(ROOTDIR)/api_docgen/Makefile.common
+
+vpath %.cmti $(ROOTDIR)/stdlib $(DOC_COMPILERLIBS_DIRS) $(DOC_STDLIB_DIRS)
+vpath %.cmt $(ROOTDIR)/stdlib
+
+odoc=odoc
```

Comment:
```suggestion
odoc=$(DOCUMENTATION_TOOL_CMD)
```

Reply:
(Done with some more handling to the case where we are in the ocamldoc mode)